core/controllers/classroom.py

```diff
@@ -32,7 +32,8 @@ class ClassroomPage(base.BaseHandler):
     def get(self):
         """Handles GET requests."""
 
-        if not constants.ENABLE_NEW_STRUCTURE_PLAYERS:
+        if not constants.ENABLE_NEW_STRUCTURE_PLAYERS or not (
+                config_domain.SHOW_CLASSROOM.value):
```

Comment:
Can you change the name to CLASSROOM_PAGE_IS_SHOWN? Constants should not start with a verb.

Reply:
Done.